test/Microsoft.AspNet.Mvc.TagHelpers.Test/ScriptTagHelperTest.cs

```diff
@@ -291,5 +431,23 @@ private TagHelperLogger<ScriptTagHelper> CreateLogger()
         {
             return new TagHelperLogger<ScriptTagHelper>();
         }
+
+        private static IHostingEnvironment MakeHostingEnvironment(IFileProvider webRootFileProvider = null)
```

Comment:
Yeah that was a carry over from `LinkTagHelper` again but never used. Will remove it.